app/src/shared/utils/constants.js

```diff
@@ -73,6 +73,7 @@ export const transactionTypes = {
 export const gasLimits = {
     DEFAULT: 3e5,
     CREATE_PRODUCT: 4e5,
+    UPDATE_PRODUCT: 4e5,
```

Comment:
This could probably be 3e5. Actual usage for me was `5e4`.

Reply:
Sure, I can change it... previously the same value (`CREATE_PRODUCT `) was used for both transactions.